fl4health/server/base_server.py

```diff
@@ -350,3 +350,69 @@ def _hydrate_model_for_checkpointing(self) -> nn.Module:
         model_ndarrays = parameters_to_ndarrays(self.parameters)
         self.parameter_exchanger.pull_parameters(model_ndarrays, self.server_model)
         return self.server_model
+
+
+class FlServerWithInitializer(FlServer):
+    initialized = False  # Add attribute
```

Comment:
Are we sure we want this to be a class attribute instead of an instance attribute?

Reply:
You're right instance attribute would be better, I just trying to avoid writing an __init__ method to keep it concise

Reply:
Yeah I feel you hahah I don't think I have ever used a class attribute. It would be nice if you could do this for instance attribute

Reply:
This is an example where that docstring parsing pr I mentioned would be useful within fl4health. The exact same length docstring occurs 3 times in this file. I think typically people would use **kwargs to avoid this, but as David has mentioned it obscures the arguments of the function.

Reply:
Added a ticket in clickup, we can discuss more details in our meeting!